test/unit/modules/helpers/get-basic-dom-node-profile-spec.js

```diff
@@ -61,16 +63,18 @@ describe('ProdPerfectRecorder.helpers.getBasicDomNodeProfile', () => {
         "ng-model": expectedP['ng-model'],
         style: `cursor: ${expectedP.cursor};`,
         title: expectedP.title,
+        unique_selector: expect.any(String),
       },
       node_name: expectedP.nodeName,
       tag_name: expectedP.tagName,
       text: expectedP.text,
       title: expectedP.title,
       type: expectedP.type,
-    })
-  })
+    });
+  });
 
   test('should exclude value from all_attrs', () => {
-    expect(getBasicDomNodeProfile(elSelect).value).toEqual(undefined)
-  })
+    expect(elSelect.value).toEqual(expect.any(String));
+    expect(getBasicDomNodeProfile(elSelect).value).toBeUndefined();
+  });
```

Comment:
I'm breaking my general practice of "one expectation per (ex)spec(tation)" here, because it's not clear reading the test that `elSelect` has a value to begin with, and this could be a false positive.

A better way of doing this would be to setup `elSelect` closer to this test, but that would require a larger refactor here and I don't think it's worth it at the moment.